lib/global-styles.php

```diff
@@ -603,6 +603,11 @@ function gutenberg_experimental_global_styles_get_editor_features( $config ) {
 		return array();
 	}
 
+	// Deprecated theme supports
+	if ( null !== get_theme_support( 'disable-custom-colors' ) ) {
+		$config['global']['features']['colors']['allowCustom'] = ! get_theme_support( 'disable-custom-colors' );
+	}
```

Comment:
This is where we added the backward compatibility support for the deprecated theme support flags.

Reply:
If the global features config is empty the previous condition will return:
```
	if (
		empty( $config['global']['features'] ) ||
		! is_array( $config['global']['features'] )
	) {
		return array();
	}
```

And in that case, we will not pass the flag value. Should we handle this case?

Reply:
Should be fixed.